src/conf.h

```diff
@@ -43,6 +43,7 @@
 #define AXEL_CONF_H
 
 typedef struct {
+	char netrc_filename[MAX_STRING];
```

Comment:
Perhaps we can add an abstraction layer here, instead of saving the name, we can save a pointer to a struct where we can later add some form of caching at least. It would also be interesting to just keep a file descriptor instead of the file name.

Reply:
I didn't understand what you meant in this case. Could you please elaborate on this point?

Reply:
Add a struct pointer there, and add everything the parser needs inside it.

Reply:
Ok.